scripts/step03_explore_examples/02_advanced_rag/src/ingest.py

```diff
@@ -76,11 +76,43 @@ def ingest_sap_docs():
     db.add_documents(chunks)
     log.success("Added SAP btp docs successfully!")
 
+def load_terraform_docs_from_github():
```

Comment:
This Function repeats the functionality, please consider moving it to [Utils](https://github.com/SAP-samples/btp-genai-starter-kit/tree/main/scripts/step03_explore_examples/utils/utils): https://github.com/SAP-samples/btp-genai-starter-kit/blob/50e3d13e0a9e9bdf2a7eff4221149986a24cab25/scripts/step03_explore_examples/01_naive_rag/library/ingest.py#L19